### DISCUSSION TITLE
Harmonize Node.js versions used by CI in all repositories

### DISCUSSION THREAD

**Discussion by marcjansen** [geostyler/geostyler#2192] [Ideas]
We should use Node.js versions 18.x and 20.x in all repos.

Here are the PRs:

* https://github.com/geostyler/geostyler/pull/2191
* https://github.com/geostyler/geostyler-openlayers-parser/pull/715
* https://github.com/geostyler/geostyler-style/pull/580
* https://github.com/geostyler/geostyler-wfs-parser/pull/517
* https://github.com/geostyler/geostyler-rest/pull/76
* https://github.com/geostyler/geostyler-sld-parser/pull/810
* https://github.com/geostyler/geostyler-shapefile-parser/pull/328
* https://github.com/geostyler/geostyler-cql-parser/pull/317
* https://github.com/geostyler/geostyler-qgis-parser/pull/477
* https://github.com/geostyler/geostyler-data/pull/281
* https://github.com/geostyler/geostyler-mapfile-parser/pull/250
* https://github.com/geostyler/geostyler-legend/pull/439
* https://github.com/geostyler/geostyler-cli/pull/351

If this agreed upon, we can merge all these.

Some action runs now produce warnings, as some actions use outdated versions, this will either be fixed in a follow-up or via mend/renovate.

**Comment by KaiVolland** [comment]
Fails for geostyler-legend. Rest ist merged.

**Reply by marcjansen** [reply]
Looking into the failing https://github.com/geostyler/geostyler-legend/pull/439

Thanks

**Comment by marcjansen** [comment]
All in now, closing this discussion.
